src/IB/IBFEMethod.cpp

```diff
@@ -1903,7 +2456,14 @@ IBFEMethod::computeStressNormalization(PetscVector<double>& Phi_vec,
                                                            surface_force_grad_var_data,
                                                            data_time,
                                                            d_lag_surface_force_fcn_data[part].ctx);
-                    Phi -= n * F_s * dA_da;
+                    if (d_scale_phi_by_J)
```

Comment:
Instead of `scale_phi_by_J`, what about something about whether phi is define with respect to reference or current coordinates?